spec/models/validation_report_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'spec_helper'
```

Comment:
I added some light specs. I'd like to do some refactoring here later, and test better.